www/src/components/Settings.tsx

```diff
@@ -85,9 +90,31 @@ const Settings = () => {
             ))}
           </Form.Control>
         </Form.Group>
+        {!showMinimizedTitle && (
+          <Form.Group>
+            <Form.Control
+              as="select"
+              value={settings.containerWidth}
+              onChange={(e: React.ChangeEvent<HTMLSelectElement>) => handleSettingsChange('containerWidth', e.target.value)}
+              floatingLabel="Container width"
+            >
+              <option value="xl">xl</option>
+              <option value="lg">lg</option>
+              <option value="md">md</option>
```

Comment:
Should there be the full set of `Container` size options here, i.e. "xs" | "sm" | "md" | "lg" | "xl"

Reply:
Added,  thanks